webpack.config.js

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable */
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
     mode: 'development',
-    entry: './src/index.js',
+    entry: {
+        index: './src/index.js',
+    },
```

Comment:
Suggestion:

I believe the brackets are unnecessary

Reply:
Thanks for the comments and suggestions. Let me make the necessary changes and submit it for re-review again. 
👍🏻